projects/plugins/boost/app/assets/src/css/main/wp-admin.scss

```diff
@@ -102,6 +102,16 @@ p {
 			color: $primary-black;
 		}
 	}
+
+	.components-panel__body-title .components-button:hover:not(:disabled,[aria-disabled=true]),
+	.components-panel__body-title .components-button[aria-expanded=true] {
+		color: inherit;
+	}
+
+	.components-panel__body-toggle.components-button:focus {
+		border-radius: inherit;
+		box-shadow: none;
+	}
```

Comment:
Can you add some comments explaining the code and why it's necessary? Do we have buttons on panel title?

Also, I have an itchy feeling about targeting aria attributes.

Reply:
I've added a comment in 5c21a04.

These are the selectors that WordPress uses to style hover/focus states.

https://github.com/WordPress/WordPress/blob/master/wp-includes/css/dist/components/style.css#L155
https://github.com/WordPress/WordPress/blob/master/wp-includes/css/dist/components/style.css#L2031